packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -16,6 +16,7 @@ public struct MediaInfo {
 public enum Capabilities: String {
     case mentions
     case unsupportedBlockEditor
+    case modalLayoutPicker
```

Comment:
If this modal layout picker is a wpcom only feature as we agreed, shouldn't this be handled only by the WPApp's feature flags only?

Reply:
This flag is handling hiding the Starter Page Template bar that currently appears on new pages. Since the wpcom feature does the same thing we don't want to prompt with the layout options twice. This just provides a way for the apps to say I've already presented a layout picker.

Reply:
In this case, shouldn't we  refer to "deactivate Starter Page Template Bar"?
It feels to me that the library shouldn't know about the external interface, but give the option to hide the internal one.

Unless I'm misunderstanding something 🤔

Reply:
I'm ok with renaming this. I read it more as the app has a modal layout picker. And once we complete this work this flag will go away as well as the current SPT so this will be relatively short-lived.

Reply:
> I read it more as the app has a modal layout picker as one of its capabilities.

Exactly. The library shouldn't need to know what capabilities the app has.
It's a bit more than just renaming, also we would need to invert it's usage here:
https://github.com/WordPress/gutenberg/blob/dd6161c291dbfa5bee7b685cc0151501fc50c3f2/packages/block-editor/src/components/page-template-picker/with-page-template-picker.native.js#L16-L18

In a way that having this capability false, the library won't show the SPT bar.

> Although once we complete this work this flag will go away as well as the current SPT so this will be relatively short-lived.

Oh I didn't know that. Then no need to be so strict about this I guess :) 
I'll leave it to you to decide and ✅